remove inline js from layout/base template

achen2401 opened the pull request:
- refactor duplicate code shared between layout.html and eproms/base.html
- remove inline js code from footer html
- remove reference to i18next in sessionMonitor.js

ivan-c commented on portal/static/js/main.js:
```diff
@@ -5428,13 +5431,95 @@ var tnthAjax = {
+        var getContent = function(cc) {
+            var content = "";
+            switch(String(cc.toUpperCase())) {
+                case "US":
```
Awesome! Wasn't sure the best way to tackle this, but this is a great start.

ivan-c commented on portal/static/js/sessionMonitor.js:
```diff
@@ -23,14 +23,14 @@ var SessionMonitorObj = function(crsf_token, session_lifetime) {
             .on("click", "#stay-logged-in", sessMon.extendsess)
             .on("click", "#log-out", sessMon.logout)
-            .find("#remaining-time").text(warningText);
+            .find("#remaining-time").text("Your session will expire in approximately {time} seconds due to inactivity.".replace("{time}",(sessMon.timeBeforeWarning / 1000)));
```
Does this need to be wrapped in `t()`?

achen2401 commented:
@ivan-c ivan, thank for taking a look at my changes. I think I addressed the issue you raised.  I encountered a js error when i was testing so I also fixed that.  let me know if you found any more issue with my changes.

achen2401 commented:
Ah finding more errors - will test before re-open this again.
